client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -318,7 +318,7 @@ public class RssSparkConfig {
 
   public static final ConfigEntry<Integer> RSS_CLIENT_BITMAP_SPLIT_NUM =
       createIntegerBuilder(new ConfigBuilder("spark.rss.client.bitmap.splitNum"))
-          .createWithDefault(1);
+          .createWithDefault(10);
```

Comment:
This is not a recommended default option. If the number is larger, the race condition of writing will increase

Reply:
@zuston can you explain more about why the write can be race？

Reply:
Oh. my fault. This will reduce the race condition, but will cost more memory, right?